networkx/readwrite/json_graph/cytoscape.py

```diff
@@ -131,9 +135,9 @@ def cytoscape_graph(data, name="name", ident="id"):
     >>> G.nodes()
     NodeView((0, 1))
     >>> G.nodes(data=True)[0]
-    {'id': '0', 'value': 0, 'name': '0'}
+    {'id': '0', 'name': '0'}
```

Comment:
I'm concerned about this... do cytoscape users who are converting data from cytoscape to networkx expect these cytoscape-specific sentinels in the resultant data? I generally agree that the information is redundant in the NetworkX data structure, but I'm not confident this won't break current users of the cytoscape conversion functions.